README.md

```diff
@@ -41,10 +41,9 @@ Configuration options are documented in [Configuration.md](helm/nginx-ingress-co
 changelog entry.
 * This will push a new git tag and trigger a new tarball to be pushed to the
 [giantswarm-catalog] and [default-catalog].
-* Update [cluster-operator] with the new version.
+* Use, test and ship the release across supported channels in new or existing WIP platform releases which have nginx IC pre-installed.
```

Comment:
```suggestion
* Test and verify the ingress controller release across supported environments in a new or existing WIP platform release.
```
`Use` sounds weird as you only will use once it is in an `active` release.

Reply:
Release CRs list included pre-installed components and used versions of those components. So with use I meant really use the released version of nginx in a new or existing/current platform WIPs, test it there, and important part is to get it shipped, shipped across all supported channels - too often do I find components/apps released but not included in any platform release or in just some, and we end up being paged and solving same problems over and over again; inconsistency make support hard, it's much easier when all clusters are on same changeset; they have less chance of being on same component changeset when component is released and used only in some supported channels; similarly it has higher chance of being used in production when there is platform release available/active using that component release; util a change is shipped and running successfully in production, IMO change is not done done.